docs/dev_cycle.intro.rst

```diff
@@ -0,0 +1,134 @@
+    
+
+We assume that your read 1-2 intros about Ethereum and the blockchain, and know Python.
+
+You don't need the complex math of the elliptic curves, but to get a grasp of the basic concepts, and the basic idea: A system that prevents bad behaviour not by moral rules, but 
+by incentives. Incentinve that make honest behaviour *more* profitable (let this bold concept sink in for a moment).
+
+Two good intros:
+
+* `A 101 Noob Intro to Programming Smart Contracts on Ethereum <https://medium.com/@ConsenSys/a-101-noob-intro-to-programming-smart-contracts-on-ethereum-695d15c1dab4>`_, from Consensys
+
+* `A Gentle Intorduction to Ethereum <https://bitsonblocks.net/2016/10/02/a-gentle-introduction-to-ethereum/>`_, from bitsonblocks
+
+And to go a little deeper:
+
+* `The Etherum Whitepaper <https://github.com/ethereum/wiki/wiki/White-Paper>`_, by Vitalik Buterin
+
+* `The Bitcoin Book <https://github.com/bitcoinbook/bitcoinbook>`_ (Some of the Bitcoin concepts are different in Ethereum, but in any case it's useful to understand the difference)
+
+
+Development Steps
+-----------------
+We will take a walk through an entire contract development cycle, with Python, Populus and Web3.py.
+
+Typical iteration will include:
+
+* Writing the contract
+* Testing, fixing bugs
+* Deployment to a local chain, make sure everything works
+* Deployment to testnet,
+* Finally deployment to mainnet which will cost real gas
+* Interaction with the contract on the blockchain.
+
+
+Glossery
+--------
+
+Just a succint reference, as a reminder if you need it during the tutorial (in a "chronological" order)
+
+**Private Key**: A long combination of alphanumeric characters. There is almost zero chance that the algorithm
+that creates this combination will create the same combination twice.
+
+**Public Key**: A combination of alphanumeric characters that is derived from the private key. It's easy to derive
+the public key from the private key, but the opposite is impossible.
+
+**Address**: A combination of alphanumeric characters that is derived from the public key.
+
+**Ethereum Account**: An address that is used on the blockchain. There are inifinite potential combinations
+of alphanumeric characters, but only when someone has the private key that the address was derived from,
+this address can be used as an *account*.
+
+**Transaction**: A message that one account sends to another. The message can contain Ether (the digital currency),
+and data. The data is used to run a contract if the account has one.
+
+**Why the private key and the public key?** The keys mechanism can confirm that the transaction was indeed authorised by the account owner, 
+that claims to sent it. 
+
+**Block**: A group of transactions
+
+**Mining**: Bundling a group of transactions into a block
+
+**Why mining is hard?** Because the miner needs to bundle transactions with additional input that requires significant
```

Comment:
Suggest removing this as it's only applicable to POW and also kind of superfluous.